### DISCUSSION TITLE
I can't get any emails from Codemagic

### DISCUSSION THREAD

**Discussion by AMBarodzich** [codemagic-ci-cd/codemagic-docs#1993] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Other - please mention in your question

### Steps to reproduce

When i try to login to Codemagic i can't get any emails from Codemagic. I can't login and i can't to get a code via email.
https://prnt.sc/BF22Kg10-E6D

### Expected results

Get a code to login

### Actual results

No any emails.
https://prnt.sc/BF22Kg10-E6D

**Comment by himesh-cm** [comment]
Hi @AMBarodzich, All systems are working fine. Can you please share your email so I can check it on my end?

**Reply by himesh-cm** [reply]
Thanks for sharing the id, I will check and get back.
FYI: i deleted the comment

**Reply by himesh-cm** [reply]
Can you please try again?

**Reply by AMBarodzich** [reply]
Thanks @himesh-cm , it works now.

**Reply by himesh-cm** [reply]
Great, 
While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!

**Reply by AMBarodzich** [reply]
Thanks a lot

**Comment by Marcelo-SmartEnvios** [comment]
Hi @himesh-cm !
I'm having the same problem, can you help me, please?
